xicam/XPCS/ingestors/__init__.py

```diff
@@ -125,7 +125,7 @@ def ingest_nxXPCS(paths):
                                       'shape': (g2.shape[0],)},
                     'g2_roi_names': {'source': source,
                                      'dtype': 'string',
-                                     'shape': []},
+                                     'shape': tuple()},
```

Comment:
why does this need to be `tuple()` instead of `[]`?

Reply:
I'm don't think it has to be. I was doing a merge at the time and ran into a conflict between those options; chose tuple because shapes are tupley

Reply:
just wondering :smile: I think that makes sense then; a tuple is more like a shape than a list since it can't grow...